### ISSUE TITLE
Improve visibility in 'Your DevPath Profile' Section in Light Mode

### ISSUE THREAD

**Issue by aspirerricha** [komalharshita/DevPath#808]
The Leaderboard, Total Points, and Badges components under the "Your DevPath Profile" section currently suffer from poor visibility due to insufficient color contrast in the light mode.

I would like to work on improving these UI components by modifying the relevant HTML and CSS files to:
->Apply distinct visual styling to the current user's row ("You") to make it stand out in the Leaderboard Section.
->Improve badge visibility with stronger color differentiation.
->Ensure improvements are consistent across both light and dark mode

The Screenshots are attatched below:

<img width="830" height="346" alt="Image" src="https://github.com/user-attachments/assets/aacaa9e2-1997-4ba8-989d-783fa37b7a3c" />
<img width="865" height="346" alt="Image" src="https://github.com/user-attachments/assets/92f358b3-0706-4c4b-8835-3a32907b6101" />

@komalharshita, Please assign this issue to me.

**Comment by Harshitabansal123** [comment]
Hi @komalharshita, I'd like to work on this issue under GSSoC 2026. Could you please assign it to me? Thank you!

**Comment by aspirerricha** [comment]
Hi, @komalharshita.I would like to work on this issue under GSSoC'26. Please assign it to me.

**Comment by aspirerricha** [comment]
It looks like this issue has already been resolved by recent changes. Since there is no remaining work to do on my side, I would like to unassign myself from this issue. Please let me know if there's any related work that still needs attention.
